DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/ViewDelegateController.cs

```diff
@@ -62,7 +62,7 @@ public IActionResult SendWelcomeEmail(int delegateId)
         {
             var delegateUser = userDataService.GetDelegateUserCardById(delegateId)!;
 
-            string baseUrl = ConfigHelper.GetAppConfig().GetAppRootPath();
+            var baseUrl = ConfigHelper.GetAppConfig().GetAppRootPath();

```

Comment:
Does this need moving to the `ConfigHelper` in the data project?

Reply:
There's a tech debt ticket ([HEEDLS-609](https://softwiretech.atlassian.net/browse/HEEDLS-609)) to resolve these remaining Web.ConfigHelper tickets. I think this should be left to be fixed and tested there rather than increasing the testing scope here. The only reason this change has been done is because the formatter automatically change to var.

Reply:
Gotcha, in that case I'm happy with the rest 😊